Author: Nathan Phillip Brink <ohnobinki@ohnopublishing.net>
Date: 2011/01/20
Purpose: Fix test compilation failure in geanyprj for out-of-source builds.
Bug: https://sourceforge.net/tracker/?func=detail&aid=3163117&group_id=222729&atid=1056532

--- a/geany-plugins/geanyprj/tests/Makefile.am
+++ b/geany-plugins/geanyprj/tests/Makefile.am
@@ -1,6 +1,6 @@
 if UNITTESTS
 include $(top_srcdir)/build/vars.build.mk
-INCLUDES = $(GEANY_CFLAGS) -I../src -DUNITTESTS
+INCLUDES = $(GEANY_CFLAGS) -I$(srcdir)/../src -DUNITTESTS
 TESTS=unittests
 noinst_PROGRAMS=unittests
 unittests_SOURCES = unittests.c ../src/utils.c
